Sources/StreamChatUI/CommonViews/AvatarView/ChatChannelAvatarView.swift

```diff
@@ -16,6 +16,23 @@ open class ChatChannelAvatarView: _View, ThemeProvider, SwiftUIRepresentable {
     open var content: (channel: ChatChannel?, currentUserId: UserId?) {
         didSet { updateContentIfNeeded() }
     }
+    
+    /// Object responsible for loading images.
+    open var imageLoader: ImageLoading {
```

Comment:
jfyi: with current syntax this will be get-only variable instead of being mutating one

Reply:
The client can override this property to provide a different implementation if needed, hence the syntax.

Reply:
we should expose a configuration for `ImageLoading